### PR TITLE
Section form

### PR DESCRIPTION
**Pull request by Nejjari001**
add form section

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -1,826 +1,860 @@
...
+  width: 273px;
+  height: 140px;
+  padding: 15px 38px 15px 16px;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #b3bac5;
+}
+
+button.btn-get {
```
Comment (1GORDON) [inline styles.css:481]: Great job team 🤝

- [ ] As specified in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md), kindly implement all button interactions for the  `Get In Touch` button.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Nejjari001 & @Ambient567 ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Nejjari001** [comment]
sorry I can't understand the requirements changes, it's about the style of
the button or position

File: styles.css (hunk 1 above)
Comment (1GORDON) [reply styles.css:481]: Hello @Nejjari001 

- [ ] The last point in the project requirements suggests that you implement button interactions of `hover` and `enable`.

**Comment by 1GORDON** [comment]
Hello @Nejjari001 

- [ ] The last point in the project requirements suggests that you implement button interactions of `hover` and `enable`.

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Nejjari001 

Your project is complete! Great job for making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- form works well
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

- [ optional] please add a PR summary indicating activities completed in the PR

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**